packages/node_modules/@ciscospark/react-component-incoming-call/src/index.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import Avatar from '@ciscospark/react-component-avatar';
+import PresenceAvatar from '@ciscospark/react-container-presence-avatar';
```

Comment:
I feel a bit weird about having a container inside a component, because it means this component will require a redux store. @bzang @moriahmaney any ideas on how we could do this while keeping the component pure?

Reply:
Good catch. I think the "right way"™ to do this is probably to pass in the Avatar as a prop from a level up.

Reply:
ok, will look into this -- should note though that other components that use PresenceAvatar import the container